docs/admin/workloads/README.md

```diff
@@ -12,16 +12,16 @@ Run:ai *Workloads* is specifically designed and optimized for AI and data scienc
 
 Runai is an open platform and supports three types of workloads each with a different set of features:
 
-1. Run:ai native workloads (trainings, workspaces, deployments)
+<!--#--> 1. Run:ai native workloads (trainings, workspaces, deployments)
       1. Submit via UI/CLI
       2. Actions - Delete/stop/connect
       3. Policies (defaults and enforcing rules)
       4. Scheduling rules
       5. RBAC
-2. Framework integrations
+<!--#-->2. Framework integrations
       1. Smart gang scheduling (workload aware)
       2. Specific workload aware visibility (GPU Utilization, workload view, dashboards)
-3. Typical Kubernetes workloads
+<!--#-->3. Typical Kubernetes workloads
```

Comment:
@lavianalon not sure I understand this comment. You want me to add the "#" to the number? This is generally not accepted.

Reply:
Sorry. I meant i think they should be displayed as subtitles and not numbered